server/test/reypayments.test.js

```diff
@@ -0,0 +1,206 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../app';
+import db from '../models/migration';
+
+const should = chai.should();
+chai.use(chaiHttp);
+
+const signupRoute = '/api/v1/auth/signup/';
+const signinRoute = '/api/v1/auth/signin/';
+let token;
+let wrongToken = 'absjesjsksksk';
+
+describe('Test case for loan repayments', () => {
+  it('should return 201 for valid credentials', (done) => {
+    chai.request(app).post(signupRoute)
+      .send({
+        firstName: 'Alice',
+        lastName: 'Williams',
+        email: 'Alice@gmail.com',
+        password: 'password',
+        work_address: 'No 12 Laogos Nigeria',
+        home_address: 'No 12 Laogos Nigeria',
+      })
+      .end((err, res) => {
+        token = res.body.data[0].token;
+        res.should.have.status(201);
+        done();
+      });
+  });
+  it(`should return 400 for user not verified`, done => {
+    chai
+      .request(app)
+      .post('/api/v1/loans')
+      .set('x-access-token', token)
+      .send({
+        amount: 100000,
+        tenor: 4
+      })
+      .end((err, res) => {
+        res.should.have.status(400);
+        res.body.error.should.equal("You are not yet verified");
+        done();
+      });
+  });
+  it('should return 200 for valid inputs user role', done => {
+    chai.request(app).post('/api/v1/users/role')
+      .send({
+        email: 'Alice@gmail.com',
+        isAdmin: 'true'
+      })
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.data.should.equal('User role updated successfully');
+        done();
+      });
+  });
+  it('Should return 200 valid email and role Admin', done => {
+    chai.request(app).patch('/api/v1/users/Alice@gmail.com/verify')
+      .set('x-access-token', token)
+      .end((err, res) => {
+        res.should.have.status(200);
+        done();
+      });
+  });
+  it(`should return 200 for valid inputs`, done => {
+    chai
+      .request(app)
+      .post('/api/v1/loans')
+      .set('x-access-token', token)
+      .send({
+        amount: 100000,
+        tenor: 4
```

Comment:
please add this payload into the mock data file. I can see it was repeated on line 86 and 87

Reply:
in fact it was repeated on a lot more lines

Reply:
I have implemented the feedback can I merge?